arbstate/inbox.go

```diff
@@ -76,15 +77,15 @@ func parseSequencerMessage(ctx context.Context, data []byte, das DataAvailabilit
 				cert, err := DeserializeDASCertFrom(bytes.NewReader(data[40:]))
 				if err != nil {
 					log.Error("Deserializing data availability cert failed", "err", err)
-					return nil
+					return nil, err
 				}
 				if err := cert.VerifyNonPayloadParts(ctx, das); err != nil { // safe because L1 verified keyset hash
 					log.Error("Invalid data availability cert", "err", err)
-					return nil
+					return nil, err
 				}
-				if cert.Timeout < maxTimestamp+7*24*60*60 {
+				if cert.Timeout < maxTimestamp+MinLifetimeSecondsForDataAvailabilityCert {
 					log.Error("Data availability cert expires too soon", "err", "")
-					return nil
+					return nil, errors.New("Data availability cert expires too soon")
```

Comment:
Will returning errors in these places halt the chain? If so we need to think through where we want to halt and where we want to ignore

Reply:
Changed this to keep the chain moving, and instead treat these error cases as batches that are valid but contain no segments.